### PR TITLE
feat: added cart page

### PR DESCRIPTION
**Pull request by paarthbajaj**
cart
empty-cart

### CODE REVIEW

**Review by psychoactiv — APPROVED** [review]

**Comment by psychoactiv** [comment]
looks good to merge

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./homepage.css" />
+    <link rel="stylesheet" href="./product-listing.css" />
+    <link rel="stylesheet" href="./cart.css" />
```
Comment (dev-enforced) [inline cart.html:9]: Keep your component library file links before css files. The css files which you made must be placed at the end.

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    />
+    <title>Cart</title>
+  </head>
+  <body>
+    <!-- *****************
+      **********************
+      ****WEBSITE HEADER****
+      **********************
+      ********************** -->
```
Comment (dev-enforced) [inline cart.html]: Keep a single line for this comment mentioning it as website-header and for other sections too

File: cart.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+@import url(https://patched-ui.netlify.app/utility-classes.css);
```
Comment (dev-enforced) [inline cart.css:1]: Getting little confused by the references in the HTML file so you can create a new file  there and have all the 3 things (components.css,styles.css and utility classes.css)

File: cart.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  color: var(--blue-shade1);
+}
+.ecom-pri-btn {
+  background-color: var(--dark-blue);
+  color: var(--white);
+  cursor: pointer;
+  border: 1px solid transparent;
+  padding: 0.4rem 0.8rem;
+  font-size: 1rem;
+  border-radius: 0.25rem;
```
Comment (dev-enforced) [inline cart.css:16]: Can use pixel here instead of rem.

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !! You can merge it Parth. Suggested some minor changes that can be taken of later.Good work on developing this component : https://deploy-preview-3--quirky-face.netlify.app/cart.html

File: cart.css (hunk 4 above)
Comment (paarthbajaj) [reply cart.css:16]: Got it for smaller units. I will use px for smaller units now on.

File: cart.css (hunk 3 above)
Comment (paarthbajaj) [reply cart.css:1]: Yeah did this randomly, I could reference in the HTML file itself. Btw, what do you suggest to use? I mean reference in HTML file or the import statement? Will make the changes.

File: cart.html (hunk 2 above)
Comment (paarthbajaj) [reply cart.html]: Ok will do it.

File: cart.html (hunk 1 above)
Comment (paarthbajaj) [reply cart.html:9]: Ok will correct it.

**Pull request merged**